packages/miniflare/src/workers/kv/assets.worker.ts

```diff
@@ -1,10 +1,14 @@
 import { SharedBindings } from "miniflare:shared";
-import { KVParams } from "./constants";
 
 interface Env {
 	[SharedBindings.MAYBE_SERVICE_BLOBS]: Fetcher;
+	__STATIC_ASSETS_REVERSE_MAP: ArrayBuffer;
```

Comment:
There should be no need for this to be an array buffer. It can just be a JSON binding to an object that maps hashes to paths and content type.